### PR TITLE
schemas: add identifiers to e-items

### PR DESCRIPTION
**Pull request by FlorianCassayre**
* identifiers schema is borrowed from documents (scheme, value, material)
* see #658

### CODE REVIEW

File: invenio_app_ils/cli.py [hunk 1]
```diff
@@ -259,6 +259,21 @@ def generate(self):
...
+                        "value": "URN:ISBN:0-395-36341-1"
+                    },
+                    {
+                        "scheme": "ISBN",
+                        "value": "0-395-36341-1",
+                        "material": "EBook"
+                    },
+                    {
+                        "scheme": "DOI",
+                        "value": ""
```
Comment (kpsherva) [inline invenio_app_ils/cli.py]: in general empty value should not happen and if it does, it should raise a ValidationError
checking the empty identifiers case should rather be as:
sometimes the records are created with `identifiers` field filled like you did here 
OR with 
```
`identifiers` = [] # or identifiers not in the record at all
```

to check how the application behaves with empty data field

File: invenio_app_ils/eitems/loaders/jsonschemas/eitems.py [hunk 2]
```diff
@@ -11,6 +11,19 @@
 from marshmallow import EXCLUDE, Schema, fields
 
 
+class IdentifierSchema(Schema):
```
Comment (kpsherva) [inline invenio_app_ils/eitems/loaders/jsonschemas/eitems.py]: I think we have the same on the document can we reuse ? ping @ntarocco

Comment (FlorianCassayre) [reply invenio_app_ils/eitems/loaders/jsonschemas/eitems.py]: That's right, it's essentially `invenio_app_ils.documents.loaders.jsonschemas.document.IdentifierSchema`.

File: invenio_app_ils/cli.py (hunk 1 above)
Comment (FlorianCassayre) [reply invenio_app_ils/cli.py]: You're absolutely right, I misunderstood what you meant.

Now that you point it out, it seems that the non-emptiness requirement of `value` is not checked by the server (and the problem is the same for document identifiers).

Comment (kpsherva) [reply invenio_app_ils/cli.py]: I think in the schema it basically thinks that empty string is allowed as a value, when it should not be

**Review by kpsherva — APPROVED** [review]

**Pull request merged**